README.md

```diff
@@ -45,7 +43,7 @@ If your project has CMake support, simply add this repo as a subdirectory:
 ```
 add_subdirectory(<path to umd>)
 ```
-You can then use `libdevice.so` by linking against the `umd_device` target wheverever is needed.
+You can then use `libdevice.so` by linking against the `umd::device` target wheverever is needed.
 ```
-target_link_libraries(tt_metal PUBLIC umd_device)
+target_link_libraries(tt_metal PUBLIC umd::device)
```

Comment:
I'm guessing most consumers should be pulling it in as a private dep, and be conscious about making it public.

```suggestion
target_link_libraries(tt_metal PRIVATE umd::device)
```